src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/readthreading/ReadThreadingAssemblerUnitTest.java

```diff
@@ -48,6 +48,17 @@ public void cleanup() throws IOException {
         seq.close();
     }
 
+    @Test
+    public void testReadThreadingAssemblerDoesntModifyInputKmerList() {
```

Comment:
add a comment with a link to the issue here so that people have some clue about the origin of this if they ever look at this test